api/v1alpha1/databasecluster_types.go

```diff
@@ -196,6 +196,8 @@ type Monitoring struct {
 type DatabaseClusterSpec struct {
 	// Paused is a flag to stop the cluster
 	Paused bool `json:"paused,omitempty"`
+	// AllowUnsafeConfiguration field used to ensure that the user can create configurations unfit for production use.
+	AllowUnsafeConfiguration bool `json:"allowUnsafeConfiguration,omitempty"`
```

Comment:
just posting this comment to prevent an accidental merging without addressing https://github.com/percona/everest-operator/pull/57#issuecomment-1702912316

Reply:
I won't merge unless we are on the same page.